### ISSUE TITLE
No sync after upgrade unison-image 2.51.2.0 and docker-sync to version 0.5.5

### ISSUE THREAD

**Issue by bjoern-flagbit** [EugenMayer/docker-sync#547]
### Error/Feature Requestion/Docs
After the upgrades there was a blank page on my nginx. I checked files and see a empty folder on the docker container. Then i used docker-sync clean and removed my containers. Pulled the new image and checked all versions. Now when i start docker-sync he is not starting correctly. 

I saw this in log:

`Error: Format string '%(ENV_APP_VOLUME)s' for 'program:unison.directory' contains names ('ENV_APP_VOLUME') which cannot be expanded. Available names: ENV_GROUP, ENV_HOME, ENV_HOSTNAME, ENV_LANG, ENV_OWNER, ENV_OWNER_HOMEDIR, ENV_OWNER_UID, ENV_PATH, ENV_PWD, ENV_SHLVL, ENV_TZ, ENV_UNISON_DIR, ENV_UNISON_EXCLUDES, ENV_VOLUME, group_name, here, host_node_name, process_num, program_name in section 'program:unison' (file: '/etc/supervisor.conf.d/supervisor.daemon.conf')`

So at the moment i have the problem that unison is not starting but maybe later i also will have the problem that the files are not synced. But i have the feeling that this error log and your last changes can be connected.

### Docker Driver
Version 17.12.0-ce-mac55 (23011)

### Sync strategy
unison (image 2.51.2.0)    

### your docker-sync.yml
```
version: "2"

options:
  verbose: true
  compose-file-path: 'docker-compose.mac.yml'
  unison_image: 'eugenmayer/unison'
  cli_mode: 'auto'

syncs:
  php-app-sync:
    notify_terminal: true
    src: './src'
    sync_strategy: 'unison'
    sync_excludes: [
      'Path .git',
      'Name .gitignore',
      'BelowPath node_modules/.bin',
      'BelowPath node_modules/angular',
      'BelowPath node_modules/angular-route',
      'BelowPath node_modules/bootstrap',
      'BelowPath node_modules/hamburgers',
      'BelowPath node_modules/requirejs',
      'BelowPath node_modules/rxjs',
      'BelowPath node_modules/symbol-observable',
      'BelowPath bower_components',
      'BelowPath sass-cache',
      'BelowPath .sass-cache',
      'BelowPath var/cache',
      'BelowPath var/page_cache',
      'BelowPath var/session',
      'BelowPath .DS_Store'
    ]
    sync_userid: '1000'
    max_attempt: 30
```

### OS
macOs HighSierra 10.13.3 (17D102)

**Comment by EugenMayer** [comment]
ensure you run docker-sync clean before you starting with the new upgraded version. This is important.

**Comment by EugenMayer** [comment]
Confirmed to be an non-clean upgrade issues. `docker-sync clean` will fix that

**Comment by bjoern-flagbit** [comment]
I did `docker-sync clean` and when i start i get this errors:

```
     command  docker run -p '127.0.0.1::5000' -v php-app-sync:/app_sync -e APP_VOLUME=/app_sync -e TZ=${TZ-`readlink /etc/localtime | sed -e 's,/usr/share/zoneinfo/,,'`} -e UNISON_SRC="-socket 5000" -e UNISON_DEST="/app_sync" -e MONIT_ENABLE="false" -e MONIT_INTERVAL="" -e MONIT_HIGH_CPU_CYCLES="" -e UNISON_ARGS="-ignore='Name Path .git' -ignore='Name Name .gitignore' -ignore='Name BelowPath node_modules/.bin' -ignore='Name BelowPath node_modules/angular' -ignore='Name BelowPath node_modules/angular-route' -ignore='Name BelowPath node_modules/bootstrap' -ignore='Name BelowPath node_modules/hamburgers' -ignore='Name BelowPath node_modules/requirejs' -ignore='Name BelowPath node_modules/rxjs' -ignore='Name BelowPath node_modules/symbol-observable' -ignore='Name BelowPath bower_components' -ignore='Name BelowPath sass-cache' -ignore='Name BelowPath .sass-cache' -ignore='Name BelowPath var/cache' -ignore='Name BelowPath var/page_cache' -ignore='Name BelowPath var/session' -ignore='Name BelowPath .DS_Store'" -e UNISON_WATCH_ARGS="" -e OWNER_UID="1000" -e HOSTSYNC_ENABLE="0" -e UNISONSOCKET_ENABLE="1"  --name php-app-sync -d eugenmayer/unison:2.51.2.0
```
**Did the OWNER_UID changed?**

And also such errors i see a lot during startup:
`  0%  00:08 ETAFailed [.docker-sync]: Error in creating directory:
Permission denied [mkdir(/app_sync/.unison..docker-sync.0366c00c04103532a9f87c5e30c41467.unison.tmp)]`

**Comment by EugenMayer** [comment]
there is indication right now, that UID mapping might be disfuncitonal, please open a new issue for that

**Comment by bjoern-flagbit** [comment]
Okay after removing `sync_userid: '1000'` and recreating everything the sync is working.
But now i need to change my config to root and that is also not a good solution.
Normally we use `www-data`. Thx for your input @EugenMayer !

**Comment by EugenMayer** [comment]
fixed with 0.5.6

**Comment by bjoern-flagbit** [comment]
Tested and it is working like expected. Thx again!
